plugin.yml

```diff
@@ -89,10 +89,11 @@ configuration:
     cache-from: [ build ]
     volumes: [ run ]
     leave-volumes: [ run ]
-    no-cache: [ build ]
+    no-cache: [ build, run ]
     use-aliases: [ run ]
     dependencies: [ run ]
     ansi: [ run ]
     tty: [ run ]
     workdir: [ run ]
     user: [ run ]
+    args: [ build, run ]
```

Comment:
this didn't exist and these don't seem to be in any particular order, so I just chucked it on the end here